packages/noya-state/src/reducers/application.ts

```diff
@@ -874,6 +971,23 @@ function accessPageLayers(
   });
 }
 
+function getNormalizedTransform(page: Sketch.Page, indexPath: IndexPath) {
```

Comment:
Maybe move these helpers to selectors? 

For `getSelectedRect`, we could probably make `state: ApplicationState` the only parameter. I don't think we have a use case for doing this on a non-selected page

Reply:
I wonder if instead of "normalized" we should call this `getLayerTransform`? I'm hesitant to call the page's coordinate system the "normalized" one, only cause I'm not sure if that's an appropriate usage of "normalized" in matrix terms. 

Also, this could be called with any parent layer and index path, not just a page, and it will still make sense: it calculates the transform of the index path parameter relative to the layer parameter.

Reply:
Ah, that makes sense. Cleaned that up so it's a little more general and better naming. Should `getNormalizedBounds` be renamed as well?